.npmignore

```diff
@@ -1 +0,0 @@
-bench
```

Comment:
Yeah, when it comes to security-related things, you're right whitelists are better. I just think that tests are part of the documentation for the package and should be published to npm. So it's easier to just ignore the `bench` folder than to whitelist every file.